.github/workflows/release-trigger.yml

```
@@ -13,11 +15,21 @@ jobs:
       - name: Save tag and SHA as an artifact for other workflows that run on workflow_run to download them
         run: |
           mkdir -p /tmp/artifacts
-          git describe --tags --exact-match  > /tmp/artifacts/metadata.txt
+          TAG=$(git describe --tags --exact-match || echo "no tag")
```

optional: remove potential for mismatch whitespace chars.
```suggestion
          TAG=$(git describe --tags --exact-match || echo "none")
```